lib/hint.rb

```diff
@@ -44,7 +44,10 @@ def which_machinery
     end
 
     def get_started(_options)
-      "You can get started by inspecting a system. Run:\n#{program_name} inspect HOSTNAME"
+      "You can get started by inspecting a system. Run:\n'#{program_name} inspect HOSTNAME'\n" \
+      "Specify user for connecting via SSH:\n" \
+      "'#{program_name} inspect --remote-user USER HOSTNAME'\n" \
+      "(Note: specified user needs sudo access or root on remote machine)"
```

Comment:
Thanks for the feedback, i basically interpreted what the ```-h``` option told me but I will implement your feedback.
Also thanks for mentioning your git commit conventions, very helpful!